overlay/files/system/oem/02_agent.yaml

```diff
@@ -6,6 +6,9 @@ stages:
        - systemctl start c3os-agent
        - systemctl enable c3os-agent
     initramfs:
+    - if: '[ ! -f "/run/cos/recovery_mode" ] && [ ! -f "/run/cos/live_mode" ]'
```

Comment:
small nit: this sounds a good candidate for a separate oem file. maybe a `_notify.yaml` where we hook 1:1 all the elemental events

Reply:
Wouldn't hooking on install mean we fire install multiple times? Ill break out this one but I don't think there are any other events we would hook on?

Reply:
my intention here is to have a common file to stream all the stages events https://rancher.github.io/elemental-toolkit/docs/customizing/stages/#stages, so it becomes an alternative to move business logic out of the cloud config (if not for system level setup of things).